src/Microsoft.Framework.Runtime/Project.cs

```diff
@@ -62,6 +63,9 @@ public string ProjectDirectory
 
         public SemanticVersion Version { get; private set; }
 
+        // Temporary while old and new runtime are separate
+        string ICompilationProject.Version { get { return Version == null ? null : Version.ToString(); } }
```

Comment:
Gotta get those C# 6 things embedded in my workflow :). Got $"", next up "?." :)